acos_client/v30/action.py

```diff
@@ -119,3 +119,26 @@ def get_vcs_summary_oper(self):
     def get_thunder_up_time(self):
         url = "/miscellenious-alb/oper"
         return self._get(url)
+
+    def probe_network_devices(self):
+        url = "/system/probe-network-devices"
+        self._post(url)
+
+    def get_acos_version(self):
+        url = "/version/oper"
+        return self._get(url)
+
+    def reload_reboot(self, acos_version=None):
```

Comment:
How do you think we change as below?
```python
    def vthunder_interface_attach(self, acos_version=None):
    """Reload/Reboot vThunder to apply the interface device changing.

    For ACOS 5.2.0 and later, it support dynamic interface detection. It allows attachment new 
    interfaces without rebooting. (notice: dynamic interface deattachment is supported in 5.2.1)
    """
```

Reply:
I think the name `vthunder_interface_attach` is not giving exact idea about the actual logic of the API definition as this API is specific for performing reload or reboot on the device depending on its ACOS version. This API helps in detecting the interface attachement by performing reload/reboot. It's not actually doing interface attachment though which is getting notified by the name `vthunder_interface_attach`.

Reply:
I can change the API name as `reload_reboot_for_interface_attach()`